source/PreferencesPanel.cpp

```diff
@@ -93,6 +93,17 @@ namespace {
 PreferencesPanel::PreferencesPanel()
 	: editing(-1), selected(0), hover(-1)
 {
+	page = Preferences::GetPreviousPage();
+	if(!page)
+		page = 'c';
```

Comment:
The previous setup for the default value should be usable here; it doesn't need changing.